templates/form-fields/recaptcha-v3-field.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Shows the `recaptcha` form field on job listing forms.
+ *
+ * This template can be overridden by copying it to yourtheme/job_manager/form-fields/recaptcha-v3-field.php.
+ *
+ * @see         https://wpjobmanager.com/document/template-overrides/
+ * @author      Automattic
+ * @package     wp-job-manager
+ * @category    Template
+ * @version     $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+$site_key = get_option( 'job_manager_recaptcha_site_key' );
+?>
+<script>
+	function onClick(e) {
```

Comment:
We should add a plugin-specific prefix to this since it's a global, eg `jm_job_submit_click`

Reply:
Done in 260ba28f0540e2a73d48dda4b4bae2e99e3e56bf